src/map/mob.c

```diff
@@ -4630,13 +4866,10 @@ void mob_load(bool minimal) {
 	}
 	sv->readdb(map->db_path, "mob_item_ratio.txt", ',', 2, 2+MAX_ITEMRATIO_MOBS, -1, mob->readdb_itemratio); // must be read before mobdb
 	mob->readchatdb();
-	if (map->db_use_sql_mob_db) {
```

Comment:
I not touched variables related to sql mob db. Will see what will left after your commits